components/login/EmailInputWIthLabel.js

```diff
@@ -1,9 +1,21 @@
-const EmailInputWithLabel = () => (
```

Comment:
Typo in file name "With" not "WIth." How do you get it to work when you imported "EmailInputWithLabel" in the sign-up and login files?